api/src/opentrons/protocol_engine/commands/calibration/move_to_location.py

```diff
@@ -3,97 +3,71 @@
+from opentrons.protocol_engine.types import CalibrationPosition
+from opentrons.types import MountType
 from opentrons.protocol_engine.commands.command import (
     AbstractCommandImpl,
     BaseCommand,
     BaseCommandCreate,
 )
 
 if TYPE_CHECKING:
-    from opentrons.protocol_engine.execution import MovementHandler
+    from opentrons.hardware_control import HardwareControlAPI
     from opentrons.protocol_engine.state.state import StateView
 
 
 MoveToLocationCommandType = Literal["calibration/moveToLocation"]
 
 
-class CalibrationPosition(str, Enum):
-    """Deck slot to move to."""
-
-    PROBE_POSITION = "probePosition"
-    ATTACH_OR_DETACH = "attachOrDetach"
-
-
-class MoveToLocationParams(PipetteIdMixin):
+class MoveToLocationParams(BaseModel):
     """Calibration set up position command parameters."""
 
     location: CalibrationPosition = Field(
         ...,
         description="Slot location to move to before starting calibration.",
     )
+    mount: MountType = Field(
+        ...,
+        description="Gantry mount to move to location.",
+    )
 
 
 class MoveToLocationResult(BaseModel):
     """Result data from the execution of a CalibrationSetUpPosition command."""
 
-    position: DeckPoint = Field(
-        ...,
-        description="Position in deck coordinates after this movement has been executed",
-    )
-
 
 class MoveToLocationImplementation(
     AbstractCommandImpl[MoveToLocationParams, MoveToLocationResult]
 ):
     """Calibration set up position command implementation."""
 
     def __init__(
-        self, movement: MovementHandler, state_view: StateView, **kwargs: object
+        self,
+        hardware_api: HardwareControlAPI,
+        state_view: StateView,
+        **kwargs: object,
     ) -> None:
-        self._movement = movement
         self._state_view = state_view
+        self._hardware_api = hardware_api
 
     async def execute(self, params: MoveToLocationParams) -> MoveToLocationResult:
         """Move the requested pipette to a given deck slot."""
-        if params.location == CalibrationPosition.PROBE_POSITION:
-            offset = DeckPoint(x=10, y=0, z=3)
-            deck_center = self._state_view.labware.get_slot_center_position(
-                DeckSlotName.SLOT_5
-            )
-            z_position = offset.z
-        else:
-            # get current z coordinate and pass it into movement destination
-            offset = DeckPoint(x=0, y=0, z=0)
-            deck_center = self._state_view.labware.get_slot_center_position(
-                DeckSlotName.SLOT_2
-            )
-            current_position = await self._movement.save_position(
-                pipette_id=params.pipetteId, position_id=None
-            )
-            z_position = current_position.position.z
-        destination = DeckPoint(
-            x=deck_center.x + offset.x, y=deck_center.y + offset.y, z=z_position
-        )
+        hardware_mount = MountType.to_hw_mount(params.mount)
```

Comment:
`to_hw_mount` is an instance method, not a class method

```suggestion
        hardware_mount = params.mount.to_hw_mount()
```